cmd/keymasterd/app.go

```diff
@@ -811,8 +813,10 @@ func (state *RuntimeState) checkAuth(w http.ResponseWriter, r *http.Request, req
 
 	}
 	if (info.AuthType & requiredAuthType) == 0 {
+		state.logger.Debugf(1, "info.AuthType: %v, requiredAuthType: %v\n",
+			info.AuthType, requiredAuthType)
 		state.writeFailureResponse(w, r, http.StatusUnauthorized, "")
-		err := errors.New("Insufficeint Auth Level")
+		err := errors.New("Insufficient Auth Level in critical section")
```

Comment:
critical section?

Reply:
I just took from the comment above. Changed.